linkml-schema/reproschema.yaml

```diff
@@ -687,3 +688,117 @@ enums:
         title: TimedOut
         description: A boolean element to describe if the response did not occur within the prescribed time.
         meaning: reproschema:TimedOut
+  InputType:
+    permissible_values:
+      radio:
+        title: radio
+        description: A radio button input type.
+        meaning: reproschema:radio
+      audioCheck:
+        title: audioCheck
+        description: An audio check input type.
+        meaning: reproschema:audioCheck
+      audioRecord:
+        title: audioRecord
+        description: An audio record input type.
+        meaning: reproschema:audioRecord
+      audioPassageRecord:
+        title: audioPassageRecord
+        description: An audio passage record input type.
+        meaning: reproschema:audioPassageRecord
+      audioImageRecord:
+        title: audioImageRecord
+        description: An audio image record input type.
+        meaning: reproschema:audioImageRecord
+      audioRecordNumberTask:
+        title: audioRecordNumberTask
+        description: An audio record number task input type.
+        meaning: reproschema:audioRecordNumberTask
+      audioRecordAudioTask:
+        title: audioRecordAudioTask
+        description: An audio record audio task input type.
+        meaning: reproschema:audioRecordAudioTask
+      audioRecordNoStop:
+        title: audioRecordNoStop
+        description: An audio record no stop input type.
+        meaning: reproschema:audioRecordNoStop
```

Comment:
is this coming from schemas and protocols? or from the UI itself?

Reply:
i see. this is fine for now. we may want to consolidate in the future.

Reply:
but this is also why I think we should provide better description and check if this still make sense and work